src/feature_extraction.py

```diff
@@ -340,6 +340,62 @@ def fit(self, X, credit_card, **kwargs):
         features['credit_card_cash_card_ratio'] = features['credit_card_drawings_atm'] / features[
             'credit_card_drawings_total']
 
+        credit_card_sorted = credit_card.sort_values(['SK_ID_CURR', 'MONTHS_BALANCE'])
```

Comment:
@pknut so this is the mean month to month difference of credit card balance? 
We could probably add say last 5 month credit balance as 5 features or maybe those diffs as 5 (or 10 idk) features

Reply:
Unfortunately after adding these new ideas (last 5 months credit balance as 5 features and an average of 5 last months diffs) our local score decrease.

Reply:
interesting, more investigation needed i guess